apps/src/lib/kits/maker/WebSerialPortWrapper.js

```diff
@@ -46,6 +48,10 @@ export default class WebSerialPortWrapper extends EventEmitter {
             this.emit('data', Buffer.from(value));
           } catch (e) {
             console.error(e);
+
+            if (e.code === DEVICE_LOST_ERROR_CODE) {
+              this.emit('disconnect');
+            }
```

Comment:
right now, i'm only disconnecting if we match on the "device lost" error code. would it be more appropriate to always disconnect in this scenario (regardless of error code)?